plugins/snmp_receiver_plugin/build/config/deinit.sh

```diff
@@ -12,6 +12,11 @@
 #
 
 set -eE
+mkdir -p /config
```

Comment:
Please remove "set -eE" since deinit.sh always return with success
Also, you don't need to create /config, /log and log file, but just run the unregister script

Reply:
removed